protocol/messenger_response.go

```diff
@@ -46,7 +45,7 @@ type MessengerResponse struct {
 	Contacts                      []*Contact
 	Invitations                   []*GroupChatInvitation
 	CommunityChanges              []*communities.CommunityChanges
-	Mailservers                   []types.Mailserver
+	StoreNodes                    []messagingtypes.StoreNode
```

Comment:
Is this not used in Desktop? If yes, then we should update it to handle the new property name

Reply:
`MessengerResponse` has `MarshalJSON` defined. I changed the type, but it is compatible (same JSON fields) and the name remains the same, so there is no breaking change for clients..
```
		StoreNodes              []messagingtypes.StoreNode          `json:"mailservers,omitempty"`
```